stylix/mk-target.nix

```diff
@@ -0,0 +1,219 @@
+/**
+  Provides a consistent target interface, minimizing boilerplate and
+  automatically safeguarding declarations related to disabled options.
+
+  # Type
+
+  ```
+  mkTarget :: AttrSet -> ModuleBody
+  ```
+
+  # Examples
+
+  This function returns the module body, not an actual module.
```

Comment:
This sentence does not seem to belong in the "Examples" section. This sentence could be used to explain the `ModuleBody` "type" in the "Type" section.